tests/atomic/main.yml

```diff
@@ -0,0 +1,527 @@
+      expected_values:
+        image_name: "{{ cockpit_cname }}"
+      expect_missing: true
+
+    - role: atomic_uninstall
+      au_image: "{{ cockpit_cname }}"
+
+    - role: atomic_images_delete
+      aid_image: "{{ ailv_match['id'] }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ cockpit_cname }}"
+      expect_missing: true
+
+    - role: atomic_containers_list_verify
+      expected_values:
+        image_name: "{{ cockpit_cname }}"
+      expect_missing: true
+
+- name: Atomic Pull With Tags
+  hosts: all
+  become: yes
+
+  tags:
+    - pull_tags
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: Set cockpit container name for RHELAH
+      set_fact:
+        fq_name: "docker.io/busybox"
+        short_name: "busybox"
+        tag: 1
+
+  roles:
+    - role: atomic_pull
+      apl_image: "{{ fq_name }}:{{ tag }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ fq_name }}"
+        tag: "1"
+
+    - role: atomic_images_delete
+      aid_image: "{{ ailv_match['id'] }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ fq_name }}"
+      expect_missing: true
+
+    - role: atomic_pull
+      apl_image: "{{ short_name }}:{{ tag }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ fq_name }}"
+        tag: "1"
+
+    - role: atomic_images_delete
+      aid_image: "{{ ailv_match['id'] }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ fq_name }}"
+      expect_missing: true
+
+- name: Atomic Pull By Digest
+  hosts: all
+  become: yes
+
+  tags:
+    - pull_digest_simple
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: Set image name
+      set_fact:
+        fq_name: "docker.io/busybox"
+        digest: "sha256:29f5d56d12684887bdfa50dcd29fc31eea4aaf4ad3bec43daf19026a7ce69912"
+
+  roles:
+    - name: atomic_pull
+      apl_image: "{{ fq_name }}@{{ digest }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ fq_name }}"
+        tag: "{{ digest|regex_replace('sha256:') }}"
+
+    # Delete the image to return to a clean state
+    - role: atomic_images_delete
+      aid_image: "{{ ailv_match['id'] }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ fq_name }}"
+      expect_missing: true
+
+- name: Atomic Pull Storage
+  hosts: all
+  become: yes
+
+  tags:
+    - pull_storage
+
+  vars_files:
+    - vars.yml
+
+  tasks:
+    - name: Set image name
+      set_fact:
+        fq_name: "docker.io/busybox"
+
+    - name: Delete all images
+      include_role:
+        name: atomic_images_delete_all
+
+    - name: Pull {{ fq_name }} using ostree
+      include_role:
+        name: atomic_pull
+      vars:
+        apl_image: "{{ fq_name }}"
+        apl_options: "--storage=ostree"
+
+    # The TYPE field doesn't show up in the json output so
```

Comment:
Is this a bug in `atomic` itself?

Reply:
I forgot to file an upstream issue for that.  I'll add a comment in the code with the issue: https://github.com/projectatomic/atomic/issues/1129